### PR TITLE
feat: Hedgehog mode skins

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

We want to offer a few fun unlocks for beta users starting with a new hedgehog skin. I went a little overboard...

## Changes

* Adds "skins" so that we can choose the set of animations from a folder of sprites. It's not quite skins in the typical sense but for our purpose close enough
* Removes the fire animation in place of "overlays" that go on top of the whole sprite
* Resets the static renderer and allows the buddy to be rendered "staticly" i.e. without movement for the MaxAI
* Some general tidying of how we do the animation rendering
* First new 🕷️ skin with some additional fun interactions...
* Easy to add keyboard based eastereggs now

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

## How did you test this code?

Lots of manual testing having too much fun

### CODE REVIEW

File: frontend/__snapshots__/scenes-other-settings--settings-user--light.png
Comment (Twixes) [inline frontend/__snapshots__/scenes-other-settings--settings-user--light.png:1]: Summat broke

**Review by Twixes — CHANGES_REQUESTED** [review]
Max went invisible

File: frontend/__snapshots__/scenes-other-settings--settings-user--light.png
Comment (benjackwhite) [reply frontend/__snapshots__/scenes-other-settings--settings-user--light.png:1]: This isn't broke, its just due to how the code works - it relys on the assets being available on us.posthog.com

File: frontend/src/lib/components/HedgehogBuddy/HedgehogOptions.tsx [hunk 1]
```diff
@@ -19,9 +22,23 @@ export function HedgehogOptions(): JSX.Element {
                 <div className="flex-1">
                     <h3>Hi, I'm Max!</h3>
                     <p>
-                        Don't mind me. I'm just here to keep you company.
-                        <br />
-                        You can move me around by clicking and dragging or control me with WASD / arrow keys.
+                        {hedgehogConfig.skin === 'spiderhog' ? (
+                            <>
+                                Well, it’s not every day you meet a hedgehog with spider powers. Yep, that's me -
+                                SpiderHog. I wasn’t always this way. Just your average, speedy little guy until a
```
Comment (Twixes) [inline frontend/src/lib/components/HedgehogBuddy/HedgehogOptions.tsx:28]: Hahahaha

File: frontend/src/lib/components/HedgehogBuddy/HedgehogOptions.tsx [hunk 2]
```diff
@@ -19,9 +22,23 @@ export function HedgehogOptions(): JSX.Element {
...
+                                radioactive spider bit me. With great power comes great responsibility so, buckle up,
```
Comment (Twixes) [inline frontend/src/lib/components/HedgehogBuddy/HedgehogOptions.tsx:29]:
```suggestion
                                radioactive spider bit me. With great power comes great responsibility, so buckle up,
```

**Review by Twixes — APPROVED** [review]

**Pull request merged**